README.md

```diff
@@ -1,6 +1,33 @@
 # Product-Portal
 
-## Installation
+## Project Details
```

Comment:
I would put the 'Project details', 'How do we complete this project?', 'ver 0.1', 'ver 0.2', 'ver 1.0', and 'mockup' in a separate file, named DETAILS.md or something, but not the README.
You can however mention the DETAILS file from the README, but this does not belong here.